### DISCUSSION TITLE
Disable GitHub pages in project language statistics

### DISCUSSION THREAD

**Discussion by danielb987** [community/community#21530] [Other Feature Feedback, Questions, & Ideas]
<p>I have a Java project where I have the project Javadoc documentation on GitHub pages. The result is that the language details statistics on the project front page is incorrect. The Javadoc pages are created automaticly by Travis so there is interest to have these files included in the language statistics.</p>
<p>On my project <a href="https://github.com/danielb987/EmojicodeEditor" title="EmojicodeEditor" rel="nofollow noopener">EmojicodeEditor</a> the language statistics show 96% html and 3% Java, and since the amount of html will grow when I add more Java code, these figures will stay more or less the same. My project is a pure Java project so I was expected Java to be close to 100%.</p>
<p>One possible solution to the problem is to only show statistics of the master branch, since all the documentation is in the gh-pages branch.</p>

**Comment by melonsong** [comment, marked as answer]
<p>Hi there! GitHub uses the open source project Linguist to determine language statistics, which supports overrides for excluding files you don’t want counted. You can add file paths to your <code>.gitattributes</code> file with the <code>linguist-documentation</code> attribute to ignore documentation files:</p>
<blockquote>
<p><code>project-docs/* linguist-documentation</code></p>
</blockquote>
<p>More detailed info can be found in <a href="https://github.com/github/linguist#overrides" rel="nofollow noopener">https://github.com/github/linguist#overrides</a>. Hope that helps!</p>

**Reply by Liquidmasl** [reply]
I am unsure if this will still get a response, but in our project it detects mainly jupyter notebook, but when I click on the language, it shows me zero found files. 
The repository is quite large, with hudreds of files and thousand lines of code, mostly python and c# 
So I cant really exclude a file, 
any ideas?
